seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/ClusterFaultToleranceIT.java

```diff
@@ -612,58 +600,106 @@ public void testStreamJobRestoreIn3NodeMasterDown() throws ExecutionException, I
 
     @SuppressWarnings("checkstyle:RegexpSingleline")
     @Test
-    @Disabled("Wait for open Imap storage")
     public void testStreamJobRestoreInAllNodeDown() throws ExecutionException, InterruptedException {
         String testCaseName = "testStreamJobRestoreInAllNodeDown";
         String testClusterName = "ClusterFaultToleranceIT_testStreamJobRestoreInAllNodeDown";
-        long testRowNumber = 1000;
+        int testRowNumber = 1000;
         int testParallelism = 6;
         HazelcastInstanceImpl node1 = null;
         HazelcastInstanceImpl node2 = null;
         HazelcastInstanceImpl node3 = null;
         SeaTunnelClient engineClient = null;
 
         try {
-            node1 = SeaTunnelServerStarter.createHazelcastInstance(
-                    TestUtils.getClusterName(testClusterName));
+            String yaml = "#\n" +
```

Comment:
Hi, Why not put it in resource directory?

Reply:
> Hi, Why not put it in resource directory?

If it is placed in the resource folder, it will take effect for the entire test case, which will cause many test cases to fail during testing.